magma/primitives/memory.py

```diff
@@ -119,16 +122,21 @@ def _get_magma_value_(self):
 class Memory(Generator2):
     def __init__(self, height, T: Kind,
                  read_latency: int = 0, read_only: bool = False,
-                 init: Optional[tuple] = None):
+                 init: Optional[tuple] = None, has_read_enable: bool = False):
         if read_latency < 0:
             raise ValueError("read_latency cannot be negative")
+        if has_read_enable and has_read_enable == 0:
```

Comment:
based on the value error should this be `read_latency == 0`? seems like either this if condition is wrong, or the error message is wrong.

Reply:
good catch